release_creation/snapshot/freeze.sh

```diff
@@ -1,2 +1,7 @@
 set -e
```

Comment:
```suggestion
#!/bin/bash -e
```

You're using bash/zsh only features, namely `[[`. Gotta specify the shell to avoid weird compatibility issues on systems that want to take this as a `sh`.